src/assets/scss/pages/story.scss

```diff
@@ -1,11 +1,36 @@
 .author-information {
+  align-items: center;
   display: flex;
   flex-direction: row;
-  margin-left: 3em;
+  flex-wrap: wrap;
+  justify-content: flex-start;
   margin-top: 0.5em;
 
+  & > * {
+    margin-right: 4rem;
+  }
+
+  .name,
+  .target-product {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+
+    .product-logo {
+      height: fit-content;
+      margin: auto 5px;
```

Comment:
it is better to set the margin or not at all, since auto is already a default value. Let's change this for `margin: 0 5px`

Reply:
Got that